src/index.js.flow

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import type { Sink, Scheduler, Source, Disposable } from "most";
+import { Stream, } from "most";
```

Comment:
Our code style uses single quotes for imports (except in TS type def files).  Let's make these single quotes.

Also, remove trailing command in the second import.